test/functional/ui/float_spec.lua

```diff
@@ -83,6 +83,28 @@ describe('floatwin', function()
     assert_alive()
   end)
 
+  it('opened with correct height', function()
```

Comment:
perhaps also write the same test for winwidth while we are at it

Reply:
Sounds good

Reply:
done